docs/markdown/Getting Started/getting-started/installation.md

```diff
@@ -6,13 +6,15 @@ createdAt: "2020-02-21T17:44:53.022Z"
 updatedAt: "2022-07-12T00:00:00.000Z"
 ---
 
-To install the `pants` binary you can use:
+You can download an installer script that will install the Pants binary with this command:
 
 ```
-/bin/bash -c "$(curl -fsSL https://static.pantsbuild.org/setup/pantsup.sh)" 
+curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/pantsup.sh
 ```
 
-This will install `pants` into `~/bin`, which must be on your PATH. The installer script will warn you if it is not.
+This script will install `pants` into `~/bin`, which must be on your PATH. The installer script will warn you if it is not.
+
+For security reasons, we don't recommend frequently curling this script directly to `bash`, e.g., on every CI run. If the script were compromised during some time window, you'd be more likely to download it during that window and be impacted. Instead, for regular use, we recommend checking this script into your repo and pointing users and CI machines to that checked-in version. The script is very simple and need not be updated very often.
```

Comment:
im just hitting this and my immediate (hacky) solution was to check the checksum - like so

```console

+    - name: Install pants
+      run: |
+        curl -sLO https://static.pantsbuild.org/setup/pantsup.sh
+        echo "a24341a70fa5a45b35e1557a1f141bb4fa4095b3f884d17a5816ae46a0bea591 pantsup.sh" |   sha256sum --check --status
+        chmod +x pantsup.sh
+        ./pantsup.sh

```

so then i thought - rather check the script in somewhere and i dont need to keep checksumming it, and ended up here

what im not totally clear about reading above advice is whether i should just check `pantsup.sh` into the repo, and then run pantsup.sh in every ci job

Or, whether i can/should customize where the pants bin is placed so it ends up in the repo rather than in `~/bin` and check that in

Reply:
ah, hmm - its a bin - probably i can cache it somewhere

im still a bit unclear about the advice/workflow here

Reply:
We're still working through the potential best practices ourselves, since this is brand new. 

I wouldn't check the pants binary in, as it's platform-specific. pantsup.sh is a think, platform-agnostic wrapper that downloads the right binary as needed. But you can, e.g., cache the downloaded binary in the CI system's cache.